addStyles.js

```diff
@@ -126,26 +128,26 @@ function removeStyleElement(styleElement) {
 	}
 }
 
-function createStyleElement(options) {
+function createStyleElement(parentElement, options) {
 	var styleElement = document.createElement("style");
 	styleElement.type = "text/css";
-	insertStyleElement(options, styleElement);
+	insertStyleElement(styleElement, parentElement, options);
```

Comment:
I believe you need to call `parentElement.ownerDocument.createElement()` instead of `document.createElement()`. One alternative is to call [adoptNode](https://developer.mozilla.org/en-US/docs/Web/API/Document/adoptNode).